### PR TITLE
Wrap closed banner messaging in span tags

### PR DESCRIPTION
**Pull request by colinswinney**
The Closed Banner messaging on polls does not include line breaks.  This PR would wrap the various messages in `<span>` tags and add additional styles so that each message would be on it's own line. 

See also this issue: https://github.com/Automattic/crowdsignal-forms/issues/185

70e3d8f5ade76eb96a4ec191a7d41b0d

### CODE REVIEW

**Review by thedebian — CHANGES_REQUESTED** [review]
Instead of wrapping messages in span, it'd be better to display a single message, in the following order of priority:
* If it's hidden, then hidden message displayed.
* If not, then if it's closed, closed message displayed
* If not, then if has voted, thanks for voting message displayed
There's no reason to say "thanks for voting" on a closed poll

**Comment by colinswinney** [comment]
That sounds good @thedebian, those changes have been addressed.

File: client/components/poll/closed-banner.js [hunk 1]
```diff
@@ -18,18 +18,18 @@ const ClosedBanner = ( {
 		'crowdsignal-forms-poll__closed-banner'
 	);
 
+	let message = '';
+	if ( isPollHidden ) {
+		message = __( 'This Poll is Hidden', 'crowdsignal-forms' );
+	} else if ( !isPollHidden && isPollClosed ) {
+		message = __( 'This Poll is Closed', 'crowdsignal-forms' );
+	} else if ( !isPollHidden && hasVoted ) {
```
Comment (thedebian) [inline client/components/poll/closed-banner.js:26]:
```suggestion
	} else if ( isPollClosed ) {
		message = __( 'This Poll is Closed', 'crowdsignal-forms' );
	} else if ( hasVoted ) {
```

Comment (thedebian) [reply client/components/poll/closed-banner.js:26]: As the first condition is isPollHidden, no need to replicate the condition for the else if

Comment (colinswinney) [reply client/components/poll/closed-banner.js]: Ahh, good call.  This has been amended.

**Review by thedebian — APPROVED** [review]

**Pull request merged**